_kb/nbt-discourse-assistantbot.md

```diff
@@ -0,0 +1,96 @@
+
+As well as helping in the formatting of motions , the bot can also fetch any of the NuNet information available in the Nu client and present it here in any public topic. Things like the current liquidity, the current difficulty, the current state of a particular motion or the current park rates can all be accessed with a simple command.
+Because the bot is hooked into the Nu Network, it can also let any user tip NuBits to any other user on this forum. To get started with tipping, send a PM with the word 'register' to [assistant](https://discuss.nubits.com/users/assistant/activity) . If you send someone who hasn't registered a tip, I'll create them an account and let them know a tip is waiting for them.
+
+To help get started, below  there is a list of the commands the bot will respond to. Please note that the bot will only respond to 'Private Message' commands via private message. Mention commands will only work when you mention its name in a forum post and type the command as shown below.
+
+## Private Message Commands
+Access these command by writing a private message to the user [assistant](https://discuss.nubits.com/users/assistant/activity)  :  
+
+ - `about` : shows bot info.
+ - `help` : shows this help screen.
+
+### Formatting Commands
+
+ - `motion hash [motion]` : Hash a motion into the preferred format. (More details below)
+ - `custodian hash [address] [amount] [text]` : Hash a Custodian Proposal into the preferred format.
+
+### Tipping Private Message Commands
+
+ - `register` : Register for tipping.
+ - `info` : Get your current balance and deposit address
+ - `history` : Get the transaction and tipping history of your tipping account
+ - `withdraw [address]` : Withdraw your entire balance from the tipping account to [address]
+ - `keywords` : Show a list of the keywords which can be used in place of an amount when tipping
+
+## Mention Commands
+
+Mention the bot in a public discussion using `@assistant`
+
+- `@assistant liquidity` : Get the current liquidity information
+- `@assistant block count` : Get the current block count
+- `@assistant block hash [index]` : Get the hash for the [index] block
+- `@assistant custodian votes` : Get the latest Custodian Votes
+- `@assistant custodian vote [address]` : Get the vote details for the Custodian address [address]
+- `@assistant motion votes` : Get the latest Motion Votes
+- `@assistant motion vote [motion_hash]` : Get the vote details for the motion with hash [motion_hash]
+- `@assistant park rates` : Get the latest Park Rate values
+- `@assistant park rate votes` : Get the latest Park Rate Votes
+- `@assistant verify` : Scan the thread for motions and custodial proposals and verify that their hashes are unchanged
+- `@assistant qrcode [address]` : Create a QR Code of the valid NBT address.
+- `@assistant tip [amount] @[user]` : Send a tip of [amount] NBT to @[user].
+
+
+## Standardise motion texts 
```

Comment:
This should probably read "Standardised Motion Texts" to keep the heading's title case in line with the rest of the site.

My American upbringing almost forced me to correct "standardised" to "standardized" but then I thought about it and came to the conclusion that I appreciate the global "flavor". However, we'll need to be careful that it doesn't distract from what we're trying to say if there are different spellings of the same word in the same document (not an issue in this case, just a comment).